exporter/loadbalancingexporter/README.md

```diff
@@ -93,6 +93,9 @@ exporters:
         - backend-2:4317
         - backend-3:4317
         - backend-4:4317
+      # Notice to config a headless service DNS in Kubernetes  
+      # dns:
+      #  hostname: otelcol-headless.observability.svc.cluster.local        
```

Comment:
I would probably suggest adding this to line 135-137 as that's where the Kubernetes-related example is located.

Reply:
Very welcome,  my friend.
Hope we can improve together.
I get your suggest, but In my understanding, both  line 135-137 and [/example/k8s-resolver's](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/loadbalancingexporter/example/k8s-resolver) are correlation about resolver of k8s.
I mean,  Is it appropriate to add DNS mode to resolver of k8s mode.

Reply:
I believe this example is correctly placed here.